src/main/java/org/mule/extension/http/api/request/builder/QueryParam.java

```diff
@@ -16,18 +16,20 @@
  * 
  * @since 1.1
  */
-public class QueryParam {
+public class QueryParam implements KeyValuePair {
```

Comment:
all parameter fields should have a jdoc. Keep in mind that jdoc will be used to auto generate documentation so it should be user oriented rather than technical.